helpdesk_mgmt/data/helpdesk_data.xml

```diff
@@ -0,0 +1,223 @@
+                          </table>
+                        </td>
+                      </tr>
+                    </tbody>
+                </table>
+                </body>
+                </html>
+            ]]>
+      </field>
+    </record>
+
+    <!-- Sequence -->
+    <record id="helpdesk_ticket_sequence" model="ir.sequence">
+      <field name="name">Helpdesk Ticket Sequence</field>
+      <field name="code">helpdesk.ticket.sequence</field>
+      <field name="prefix">HT</field>
+      <field name="padding">5</field>
+      <field name="company_id"/>
+    </record>
+    <!-- Stages -->
+    <record id="helpdesk_ticket_stage_new" model="helpdesk.ticket.stage">
+      <field name="sequence">1</field>
+      <field name="name">New</field>
+      <field name="unattended">True</field>
+      <field name="closed">False</field>
+      <field name="company_id"/>
+    </record>
+    <record id="helpdesk_ticket_stage_in_progress" model="helpdesk.ticket.stage">
+      <field name="sequence">2</field>
+      <field name="name">In Progress</field>
+      <field name="unattended">False</field>
+      <field name="closed">False</field>
+      <field name="company_id"/>
+    </record>
+    <record id="helpdesk_ticket_stage_awaiting" model="helpdesk.ticket.stage">
+      <field name="sequence">3</field>
+      <field name="name">Awaiting</field>
+      <field name="unattended">False</field>
+      <field name="closed">False</field>
+      <field name="company_id"/>
+    </record>
+    <record id="helpdesk_ticket_stage_done" model="helpdesk.ticket.stage">
+      <field name="sequence">3</field>
+      <field name="name">Done</field>
+      <field name="unattended">False</field>
+      <field name="closed">True</field>
+      <field name="fold">True</field>
+      <field name="mail_template_id" ref="helpdesk_mgmt.closed_ticket_template"/>
+      <field name="company_id"/>
+    </record>
+    <record id="helpdesk_ticket_stage_cancelled" model="helpdesk.ticket.stage">
+      <field name="sequence">4</field>
+      <field name="name">Cancelled</field>
+      <field name="unattended">False</field>
+      <field name="closed">True</field>
+      <field name="fold">True</field>
+      <field name="mail_template_id" ref="helpdesk_mgmt.closed_ticket_template"/>
+      <field name="company_id"/>
+    </record>
+    <!-- Channels -->
+    <record id="helpdesk_ticket_channel_web" model="helpdesk.ticket.channel">
+      <field name="name">Web</field>
+    </record>
+    <record id="helpdesk_ticket_channel_email" model="helpdesk.ticket.channel">
+      <field name="name">Email</field>
+    </record>
+    <record id="helpdesk_ticket_channel_phone" model="helpdesk.ticket.channel">
+      <field name="name">Phone</field>
+    </record>
+    <record id="helpdesk_ticket_channel_other" model="helpdesk.ticket.channel">
+      <field name="name">Other</field>
+    </record>
+    <!--default alias for leads-->
+    <record id="mail_alias_lead_info" model="mail.alias">
```

Comment:
You can not use this id "mail{alias_lead_info" because it is used on crm module, and this do that module crm will be uninstallable.